src/Components/Dialogs/GetPerson/getPerson.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import style from './getPerson.module.css';
+
+const GetPerson = (props) => {
+  const { items } = props;
+  const listItems = Object.keys(items).map((elem) => <div><NavLink to={`/dialogs/${elem}`} className={style.item}>{items[elem].name}</NavLink></div>);
```

Comment:
Additionally, I suggest you to store your route strings ('/dialog/') in a central constants file somewhere at the root level.
This way you will have just one place to change the path to some parts of your website ('/dialog/' => '/dialogs/').
Also, in the case of this line the implementation of the route could be the following
`... to={`${DIALOG_ROUTE}${elem}`} ...`

Reply:
Sure.I added new elem at constans.